app/src/sharedTest/java/org/oppia/android/app/parser/HtmlParserTest.kt

```diff
@@ -141,8 +137,7 @@ class HtmlParserTest {
 
   @After
   fun tearDown() {
-    // Ensure lingering tasks are completed.
-    testCoroutineDispatchers.advanceUntilIdle()
+//    testCoroutineDispatchers.unregisterIdlingResource()
```

Comment:
Please remove this since it's commented out & appears to not be needed.

Reply:
Removed unwanted code.